pkgs/development/libraries/opencv/3.x.nix

```diff
@@ -15,6 +15,7 @@
 , enableFfmpeg    ? false, ffmpeg
 , enableGStreamer ? false, gst_all_1
 , enableEigen     ? false, eigen
+, enableOpenblas  ? false, openblas
```

Comment:
The default implementation of blas/lapack is not optimized. Using openblas (or atlas) greatly improves performance. 

Ceres-solver may also benefit from this (http://ceres-solver.org/installation.html#dependencies):
> On UNIX OSes other than Mac OS X we recommend ATLAS, which includes BLAS and LAPACK routines. It is also possible to use OpenBLAS.